src/components/v5/frame/ColonyHome/partials/Members/Members.tsx

```diff
@@ -19,10 +19,11 @@ const Members = () => {
     totalMembers: members,
     totalMemberCount,
     loading: membersLoading,
+    filteredMembers,
   } = useMemberContext();
```

Comment:
Not your changes, but this and related contexts are perfect candidates for a refactor. There's `totalMembers`, `filteredMembers` that need to be filtered further, I cannot wrap my head around what's happening there